decent-dao/fractal-interface
# Feature |  Import Existing Token (Create Workflow)

## Description
This PR adds the ability to import an existing ERC20 token and wrap it for use as a Governance token in a DAO. It covers the first part of deploying a new DAO with this new feature and updates the UI during the token configuration step.

## Main changes
- Modified `AzoriusTxBuilder.ts` to handle the import and wrapping of existing ERC20 tokens.
- Updated `AzoriusTokenDetails.tsx` to include new UI elements for the token configuration step.

## Additional Notes
- Fixed an issue with Toasts not properly closing and a missing translation.
- This PR also covers checking for tokens that are already presumed to be ERC20 Votes tokens.

## Testing
1. Create a new Token DAO using the following methods:
   - As normal
   - With existing VotesToken
   - With existing ERC20 token
2. Ensure that the DAO deploys correctly and shows up as a Token Voting DAO. Note that there is no other way to ensure proper deployment until the other end of this feature is worked on.

## Screenshots (if applicable)
![localhost_3000_create](https://user-images.githubusercontent.com/38386583/233542977-c9a9867d-e816-4368-87bf-469b31683a73.png)
